src/components/BarChart.tsx

```diff
@@ -0,0 +1,30 @@
+import Plot from 'react-plotly.js';
+
+import data from '../fake-data/fake_data_agg.json'
+
+const CountLocation = data.data_classification.filter(data => data.data_type == 'Location').length
+const CountBehavioural = data.data_classification.filter(data => data.data_type == 'Behavioural').length
+const CountCommunication = data.data_classification.filter(data => data.data_type == 'Communication').length
+const CountTechnical = data.data_classification.filter(data => data.data_type == 'Technical').length
+const CountSocialRelationship = data.data_classification.filter(data => data.data_type == 'Social relationship').length
+const CountContact = data.data_classification.filter(data => data.data_type == 'Contact').length
+const CountTransactional = data.data_classification.filter(data => data.data_type == 'Transactional').length
+
+console.log(CountTransactional)
```

Comment:
what's the purpose of this line ?

Reply:
It was written for test purpose. Well caught, I've removed it.